api/src/controllers/kots/KotsAPI.ts

```diff
@@ -316,20 +317,25 @@ export class KotsAPI {
       password = body.password;
     }
 
+    const liveness = new Repeater(() => {
+      return new Promise((resolve) => {
```

Comment:
would it make more sense to create the promise inside the repeater?

Reply:
irl'ed